.github/workflows/ci-phpunit.yml

```
@@ -11,11 +11,12 @@ on:
 
 jobs:
   phpunit-tests:
+    name: PHP ${{ matrix.php }} Latest
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        php-versions: [ '7.4','8.0','8.1','8.2' ]
+        php: [ '7.4','8.0','8.1','8.2' ]
```

Slight preference for consistency:
```suggestion
        php-version: [ '7.4','8.0','8.1','8.2' ]
```

---

Done